### PR TITLE
Database is added

### PR DESCRIPTION
**Pull request by Songsterr99**

### CODE REVIEW

**Comment by Songsterr99** [comment]
done

File: ITechArt.StudentsLab.Database/dbo/Tables/FeedbackAnswers.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE [dbo].[FeedbackAnswers] (
+    [Id]         INT            IDENTITY (1, 1) NOT NULL,
+    [Answer]     NVARCHAR (MAX) NOT NULL,
+    [MentorId]   INT            NOT NULL,
```
Comment (AndreiShenets) [inline ITechArt.StudentsLab.Database/dbo/Tables/FeedbackAnswers.sql]: Uniquie constraint

File: ITechArt.StudentsLab.Database/dbo/Tables/LabRoles.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿CREATE TABLE [dbo].[LabRoles] (
+    [Id]     INT IDENTITY (1, 1) NOT NULL,
+    [UserId] INT NOT NULL,
```
Comment (AndreiShenets) [inline ITechArt.StudentsLab.Database/dbo/Tables/LabRoles.sql]: Unique constraint

File: ITechArt.StudentsLab.Database/dbo/Tables/LabRoles.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    [RoleId] INT NOT NULL,
+    [LabId]  INT NOT NULL,
+    CONSTRAINT [PK_LabRoles] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_LabRoles_To_Roles] FOREIGN KEY ([RoleId]) REFERENCES [dbo].[Roles] ([Id]),
+    CONSTRAINT [FK_LabRoles_To_Users] FOREIGN KEY ([UserId]) REFERENCES [dbo].[Users] ([Id])
+);
+
+
+GO
+CREATE NONCLUSTERED INDEX [IX_LabsRoles]
```
Comment (AndreiShenets) [inline ITechArt.StudentsLab.Database/dbo/Tables/LabRoles.sql]: Remove index

File: ITechArt.StudentsLab.Database/dbo/Tables/Labs.sql [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Labs] (
+    [Id]        INT           IDENTITY (1, 1) NOT NULL,
+    [Name]      NVARCHAR (50) NOT NULL,
+    [StartDate] DATE          NOT NULL,
+    [EndDate]   DATE          NOT NULL,
+    [CityId]    INT           NOT NULL,
+    [TypeId]    INT           NOT NULL,
+    [isVisible] BIT           NOT NULL,
```
Comment (AndreiShenets) [inline ITechArt.StudentsLab.Database/dbo/Tables/Labs.sql]: Rename to IsDraft or IsActive

File: ITechArt.StudentsLab.Web/ITechArt.StudentsLab.Web.csproj [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Controllers\" />
+    <Folder Include="View\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="wwwroot\favicon.png" />
```
Comment (AndreiShenets) [inline ITechArt.StudentsLab.Web/ITechArt.StudentsLab.Web.csproj]: These files shouldn't be added

**Comment by Songsterr99** [comment]
done v3.0

**Pull request merged**